ios/FluentUI.Tests/DatePickerControllerTests.swift

```diff
@@ -84,6 +84,22 @@ class DatePickerControllerTests: XCTestCase {
             return
         }
     }
+
+    // MARK: - helper functions
+
+    func getNextIndexPath(indexPath: IndexPath) -> IndexPath {
+        if indexPath.item == 6 {
```

Comment:
Looks like you could probably `% 7` here, but I'm curious if this is something that should actually be fixed in the control, rather than the test?

Reply:
Yes! That was my first thought as well but in swift -1%7 = -1 so while it'd work for `getNextIndexPath`, it wouldn't for `getPreviousIndexPath`. Would you prefer I just do it for `getNextIndexPath` or keep as is for consistency. But I can move this to the control.

Reply:
Hm. I guess I can see why -1 would be 7*0 + -1 instead of 7*-1 + 6. I still think this is probably a fix that needs to happen in the control, rather than in the test, so I'll look into it some more.